WARNING: fsync() not supported in selected backend.  Using value of 0.
IOR-3.4.0+dev: MPI Coordinated Test of Parallel I/O
Began               : Sun Dec  6 00:39:50 2020
Command line        : ./ior -Q 1 -F -g -G 4711 -z --random-offset-seed=11 -e -o ./datafiles/2020.12.06-00.26.45/ior-rnd/file -O stoneWallingStatusFile=./results/2020.12.06-00.26.45/ior-rnd.stonewall -O stoneWallingWearOut=1 -t 4096 -b 4096 -s 1000000 -w -D 300 -a S3-libs3 --S3-libs3.host=s3.eu-de.cloud-object-storage.appdomain.cloud:443 --S3-libs3.secret-key=foo --S3-libs3.access-key=foo --S3-libs3.s3-compatible --S3-libs3.use-ssl -k
Machine             : Linux mg100
WARNING: S3 S3_init:509 (path:ior96) "ErrorBucketAlreadyExists": The requested bucket name is not available. The bucket namespace is shared by all users of the system. Please select a different name and try again. 
TestID              : 0
StartTime           : Sun Dec  6 00:39:50 2020
Path                : ./datafiles/2020.12.06-00.26.45/ior-rnd/file.00000000
FS                  : 0.0 GiB   Used FS: 0.0%   Inodes: 0.0 Mi   Used Inodes: -inf%

Options: 
api                 : S3-libs3
apiVersion          : 0.5
test filename       : ./datafiles/2020.12.06-00.26.45/ior-rnd/file
access              : file-per-process
type                : independent
segments            : 1000000
ordering in a file  : random
ordering inter file : no tasks offsets
nodes               : 2
tasks               : 2
clients per node    : 1
repetitions         : 1
xfersize            : 4096 bytes
blocksize           : 4096 bytes
aggregate filesize  : 7.63 GiB
stonewallingTime    : 300
stoneWallingWearOut : 1

Results: 

access    bw(MiB/s)  IOPS       Latency(s)  block(KiB) xfer(KiB)  open(s)    wr/rd(s)   close(s)   total(s)   iter
------    ---------  ----       ----------  ---------- ---------  --------   --------   --------   --------   ----
stonewalling pairs accessed min: 8718 max: 8766 -- min data: 0.0 GiB mean data: 0.0 GiB time: 300.0s
WARNING: Expected aggregate file size       = 8192000000
WARNING: Stat() of aggregate file size      = 0
WARNING: Using actual aggregate bytes moved = 71811072
WARNING: Maybe caused by deadlineForStonewalling
write     0.226947   58.10      300.08      4.00       4.00       0.030345   301.73     0.003990   301.76     0   
WARNING: S3 S3_final:521 (path:ior96) "ErrorBucketNotEmpty": The bucket you tried to delete is not empty. 
